libs/blog/layouts/ui-navigation/src/lib/navigation/navigation.component.ts

```diff
@@ -33,6 +33,7 @@ export type NavItem = {
 })
 export class NavigationComponent {
   readonly layout = input<'vertical' | 'horizontal'>('horizontal');
+  readonly navFooter = input<boolean>(false);
```

Comment:
Maybe we could name this not by context, but by it's meaning (which is kind of hard to name, as I understand it makes text white)